.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+### What happened?
+    Tell us what happened and provide as much details as possible, including logs.
+
+### What did you expect to happen?
+    Tell us about expected behaviour.
+
+### What is the GPU model?
+    Tell us about hardware configuration of the GPU.
+
+### What is the environment?
+    Is the DCGM exporter bare-metal or cloud provider? Do you run the DCGM exporter on Kubernetes or maybe as a standalone Docker container?
+
+### How did you deploy the dcgm-exporter and what is the configuration?
+    Tell us how did you deploy the dcgm-exporter. Did you used helm? Did you build from source? Maybe, the dcgm-exporter
+    was deployed by the gpu-operator?
```

Comment:
"Tell us how you deployed DCGM-Exporter. Did you use helm, build from source or use the GPU Operator?"